jest.config.js

```diff
@@ -12,15 +12,15 @@ module.exports = {
     ],
     testURL: 'http://localhost',
     transformIgnorePatterns: ['<rootDir>/node_modules/'],
-    verbose: false,
+    verbose: true,
```

Comment:
Hi @davidokonji What does this do please?

Reply:
@Chiazokam needed the outputs of the test to be shown in order to properly track which test block fails, see example below 
<img width="1160" alt="Screenshot 2019-05-13 at 09 54 53" src="https://user-images.githubusercontent.com/30610099/57608632-3900a100-7565-11e9-95bd-9e18e8fb85ae.png">

Reply:
Cool